keyboards/binepad/BNR1/config.h

```diff
@@ -0,0 +1,95 @@
+/* Copyright 2020 BINEPAD
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4249 // Binepad
+#define PRODUCT_ID      0x4212
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Binepad
+#define PRODUCT         BNR1
+
+
+#define VIAL_KEYBOARD_UID {0xFB, 0x52, 0x5D, 0x17, 0xF3, 0x0B, 0xA3, 0x92}
```

Comment:
```suggestion
```

Reply:
VIAL is not supported in this repo, and as such, the configuration needs to be removed. 
```suggestion
```